controller/src/main/java/org/jboss/as/controller/ModelControllerImpl.java

```diff
@@ -1731,25 +1286,17 @@ public void close() throws IOException {
 
         T fromOperationResponse(OperationResponse or);
 
-        ResponseConverter<ModelNode> TO_MODEL_NODE = new ResponseConverter<ModelNode>() {
-            @Override
-            public ModelNode fromOperationResponse(OperationResponse or) {
-                ModelNode result = or.getResponseNode();
-                try {
-                    or.close();
-                } catch (IOException e) {
-                    ROOT_LOGGER.debugf(e, "Caught exception closing %s whose associated streams, "
-                            + "if any, were not wanted", or);
-                }
-                return result;
+        ResponseConverter<ModelNode> TO_MODEL_NODE = or -> {
+            ModelNode result = or.getResponseNode();
+            try {
+                or.close();
+            } catch (IOException e) {
+                ROOT_LOGGER.debugf(e, "Caught exception closing %s whose associated streams, "
+                        + "if any, were not wanted", or);
             }
+            return result;
```

Comment:
ah, I forgot I did that. IDE tempted me :) 
I can revert this change if you don't feel comfortable with it.